qt/python/mantidqt/mantidqt/project/plotsloader.py

```diff
@@ -305,7 +305,7 @@ def update_properties(self, ax, properties):
         if properties["bounds"]:
             ax.set_position(properties["bounds"])
         ax.set_navigate(properties["dynamic"])
-        ax.axison = properties["axisOn"]
+        ax.axisOn = properties["axisOn"]
```

Comment:
```suggestion
        ax.axison = properties["axisOn"]
```
I think the property is `axison`, e.g. https://github.com/matplotlib/matplotlib/blob/a254b687df97cda8c6affa37a1dfcf213f8e6c3a/lib/matplotlib/axes/_base.py#L3495